cmd/node/multi_node.go

```diff
@@ -131,6 +131,7 @@ func getTestDefaultConfig() *config.Config {
 		log.Error("cannot load config from file")
 		return nil
 	}
+	cfg.P2P.SwarmConfig.RandomConnections = 0
```

Comment:
this needs to be configured, because simulator returns closed channel for initial discovery and this behaviour needs to be preserved for tests to pass